### ISSUE TITLE
Docs - replace python with python3 for intersphinx

### ISSUE THREAD

**Issue by stevepiercy** [Pylons/pyramid#2429]
Our intersphinx links and many other links to Python docs point to version 2.x, e.g.,

```rst
:ref:`Python Packages <python:tut-packages>`
```

We should be using this:

```rst
:ref:`Python Packages <python3:tut-packages>`
```

...where we edit `docs/conf.py` to generate intersphinx links to the Python 3.x docs concurrently with Python 2:

```python
    'python': (
        'http://docs.python.org/2',
        None),
    'python3': (
        'https://docs.python.org/3/',
        None),
```

Some links might change between Python 2 and 3, so each rendered link needs to be tested.

Assuming all links to Python 2 can be changed to 3, then `docs/conf.py` can drop the intersphinx entry for 'python' as the final step of this task.

I'm already doing this for the branch [docs/easy-install-to-pip.2104](https://github.com/Pylons/pyramid/tree/docs/easy-install-to-pip.2104) and issue #2223, but there are other docs that are outside the scope of that branch/issue that will also need review.

**Comment by mmerickel** [comment]
I would just re-alias python to python3's url and add a python2 if anything needs to ref py2 explicitly.

**Comment by stevepiercy** [comment]
When I did this:

```python
 'python': (
        'https://docs.python.org/3',
        None),
```
and
```rst
:ref:`Python Packages <python:tut-packages>`
```
Links still pointed at Python 2 docs. I even blasted away the build directory. Do you mean something else?

**Comment by mmerickel** [comment]
That seems like something is still sticking around for whatever reason.

**Comment by mmerickel** [comment]
@stevepiercy don't start tagging this pip work on the 1.7 milestone unless you're positive it's getting done in the next few weeks.

**Comment by stevepiercy** [comment]
Unless something untoward should happen to me, it will.

**Comment by stevepiercy** [comment]
I couldn't figure out why things stuck around. Were you able to replicate the issue? Could it be some obscure default in intersphinx?

**Comment by mmerickel** [comment]
I'm pretty anti changing every reference in the doc to `python3:` so if we can't figure this out then I opt to keeping things at python2 until we can. Version-specific directives are a pain to maintain.

**Comment by stevepiercy** [comment]
Were you able to replicate the issue? I'd like to know if it's just my setup. If you can replicate it, then I'll try a clean virtualenv to eliminate any possible stickiness.

**Comment by stevepiercy** [comment]
It was me. There's a stray `docs/quick_tutorial/conf.py` and I thought I was editing `docs/conf.py`. argh.

**Comment by stevepiercy** [comment]
Closed by #2431
